### DISCUSSION TITLE
New-HTMLChart, any way to display the values on the chart instead of the percentage?

### DISCUSSION THREAD

**Discussion by tomerikko** [EvotecIT/PSWriteHTML#386] [Q&A] [answered]
in New-HTMLChart, is there any way to display the values on the chart? (not the the percentage)

BTW, thanks a lot for creating this plug in, i find it amazing!!!!

**Comment by tomerikko** [comment]
@PrzemyslawKlys    any suggestions are much appreciated.

**Reply by PrzemyslawKlys** [reply, marked as answer]
There's no option to change it at the moment. If you hover over it it will show you values.

**Reply by tomerikko** [reply]
OK,  thanks for confirming. appreciated.
If possible, i would love if we can add this to the requested list for future versions.
